src/Resources/contao/helper/functions.php

```diff
@@ -15,9 +15,9 @@
  * @param string $strMessage
  * @param string $strLog
  */
-function log_message($strMessage, $strLog='error.log')
```

Comment:
Ah, I see. I've been asking myself this for a long time already. What does SF2 itself do if there occur errors before the container is ready?